perforce.js

```diff
@@ -51,15 +56,15 @@ function activate() {
 				if(config.addOnFileCreate) {
 					_watcher.onDidCreate(w_onFileCreated);
 				}
-			}
+			}			
 		});
 	}
 }
 exports.activate = activate;
 
 function buildCmdline(command, args)
 {
-	var cmdline = "p4";
+	var cmdline = _cmdline;
 	if (isWin) {
 		cmdline += ".exe";

```

Comment:
That part was already there. I agree that there might be a better solution but I think it's outside the scope of this PR